src/plugins/mer/mer.pro

```diff
@@ -28,7 +28,7 @@ FORMS += \
     merrpmvalidationstepconfigwidget.ui \
     merrunconfigurationaspectqmllivedetailswidget.ui \
     mersdkdetailswidget.ui \
-    mersdkselectiondialog.ui \
+    mersdkselectiondialog.ui
```

Comment:
Please preserve the trailing backslash - it allows to append items without the need to touch the last item. More occurrences below.